sql/parser/sql.y

```diff
@@ -349,7 +349,7 @@ import "github.com/cockroachdb/cockroach/sql/privilege"
 %token <str>   LEADING LEAKPROOF LEAST LEFT LEVEL LIKE LIMIT LISTEN LOAD LOCAL
 %token <str>   LOCALTIME LOCALTIMESTAMP LOCATION LOCK LOCKED LOGGED LSHIFT
 
-%token <str>   MAPPING MATCH MATERIALIZED MAXVALUE MINUTE MINVALUE MODE MONTH MOVE
+%token <str>   MAPPING MATCH MATERIALIZED MAXVALUE MICROSECOND MILLISECOND MINUTE MINVALUE MODE MONTH MOVE
```

Comment:
I think you also have to add `MICROSECOND` and `MILLISECOND` to the `keywords` map in `keywords.go`. I'm surprised/confused that parsing works. Oh, perhaps because it is treating these through the `SCONST` rule.